test/local-conf-templates/conf/azkaban.properties

```diff
@@ -0,0 +1,55 @@
+# Azkaban Personalization Settings
+azkaban.name=Local
+azkaban.label=My Local Azkaban
+azkaban.color=#FF3601
+web.resource.dir=azkaban-web-server/build/install/azkaban-web-server/web
+default.timezone.id=America/Los_Angeles
+# Azkaban UserManager class
+user.manager.class=azkaban.user.XmlUserManager
+user.manager.xml.file=azkaban-solo-server/src/main/resources/conf/azkaban-users.xml
+# Loader for projects
+executor.global.properties=azkaban-solo-server/src/main/resources/conf/global.properties
+azkaban.project.dir=projects
+azkaban.execution.dir=executions
+# DB
+database.sql.scripts.dir=azkaban-db/src/main/sql
+database.check.version=true
+database.auto.update.tables=true
+#mysql
+database.type=mysql
+mysql.port=3306
+mysql.host=localhost
+mysql.database=test
+mysql.numconnections=100
+mysql.user=
+mysql.password=
+# h2 db
+#database.type=h2
+#h2.path=data/azkaban
+#h2.create.tables=true
+# load the commonprivate.properties
+azkaban.jobtype.plugin.dir=azkaban-solo-server/src/main/resources
+# Velocity dev mode
+#velocity.dev.mode=false
+velocity.dev.mode=true
+# Azkaban Jetty server properties. Ignored in tomcat
+jetty.use.ssl=false
+jetty.ssl.port=8043
+jetty.maxThreads=25
+jetty.port=8081
+# Azkaban Executor settings
+executor.port=12321
+executor.maxThreads=50
+executor.flow.threads=30
+# mail settings
```

Comment:
Now that we don't specify and use mail configs in solo server, should we just remove the 4 lines of email parameters.

Reply:
What does it mean that "we don't specify and use mail configs in solo server"? I would find it handy to be able to test the mail settings for testing as well..

Reply:
👍